astronomer/providers/microsoft/azure/example_dags/example_adf_run_pipeline.py

```diff
@@ -5,9 +5,7 @@
 
```

Comment:
change dag id to `dag_id="example_async_adf_run_pipeline"` to be consistent with other async DAG examples